### PR TITLE
Adjust code regarding the new design tokens

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
As a part of https://github.com/greenpeace/planet4-master-theme/pull/2066

- Mostly replace div's description to paragraphs to make it more semantic
- Fix gallery block

Using the [Umbriel's instance](https://github.com/greenpeace/planet4-test-umbriel/blob/main/composer-local.json#L6-L7) as a demo page.

### CODE REVIEW

File: assets/src/blocks/Gallery/GalleryFrontend.js [hunk 1]
```diff
@@ -49,12 +49,16 @@ export const GalleryFrontend = ({
...
       }
       {attributes.gallery_block_description &&
-        <div className="page-section-description" dangerouslySetInnerHTML={{__html: attributes.gallery_block_description}} />
+        <p className="page-section-description" dangerouslySetInnerHTML={{__html: attributes.gallery_block_description}} />
       }
 
-      {layout === 'slider' && <GalleryCarousel onImageClick={openLightbox} images={images} />}
-      {layout === 'three-columns' && <GalleryThreeColumns onImageClick={openLightbox} images={images} postType={postType} />}
-      {layout === 'grid' && <GalleryGrid onImageClick={openLightbox} images={images} />}
+      {images.length ? (
```
Comment (GP-Dan-Tovbein) [inline assets/src/blocks/Gallery/GalleryFrontend.js:55]: I added this to avoid render components if the length of images is empty.

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [telesto](https://www-dev.greenpeace.org/test-telesto) | [admin](https://www-dev.greenpeace.org/test-telesto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-telesto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-telesto) | [composer-local.json](https://github.com/greenpeace/planet4-test-telesto/blob/main/composer-local.json)

:watch: 2023.07.12 13:10:48

File: assets/src/blocks/Gallery/GalleryFrontend.js (hunk 1 above)
Comment (mleray) [reply assets/src/blocks/Gallery/GalleryFrontend.js:55]: Maybe we should hide the whole block if there are no images? 🤔

**Review by mleray — APPROVED** [review]
Tested on local, looks good to me! I only left a small comment that is not a blocker.

File: assets/src/blocks/Gallery/GalleryFrontend.js (hunk 1 above)
Comment (GP-Dan-Tovbein) [reply assets/src/blocks/Gallery/GalleryFrontend.js:55]: Sounds logic! But there is a text and description also included

**Pull request merged**